feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,292 @@
+
+## Procedure
+
+### Test environment setup
+
+*   DUT has 5 ports connected to ATE.
+
+    ```
+                              -------
+                             |       | ==== LAG1 ==== [ ATE:Port2, ATE:Port3 ]
+          [ ATE:Port1 ] ---- |  DUT  |
+                             |       | ==== LAG2 ==== [ ATE:Port4, ATE:Port5 ]
+                              -------
+    ```
+
+*   Routes are advertised from ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5.
+*   Traffic is generated from ATE:Port1.
+*   ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5 are used as the destination port for GUE encapsulated traffic.
+
+#### Configuration
+
+1.  DUT:Port1 is configured as Singleton IP interface towards ATE:Port1.
+
+2.  DUT:Port2 and DUT:Port3 are configured as LAG1 IP interface towards
+    ATE:Port2 and ATE:Port3 respectively.
+
+3.  DUT:Port4 and DUT:Port5 are configured as LAG2 IP interface towards
+    ATE:Port4 and ATE:Port5 respectively.
+
+4.  DUT is configured to form two eBGP sessions with ATE using the directly
+    connected LAG interface IPs.
+
+5.  ATE:LAG1 and ATE:LAG2 are configured to advertise destination networks
+    (IPv4-DST-NET, IPv6-DST-NET) and tunnel destination (IPv4-DST-GUE) to DUT.
+    The protocol next hops for the destination networks should be re-configured
+    as below:
+
+    -   Destination network IPv4-DST-NET with protocol next hop PNH-IPv4.
+    -   Destination network IPv6-DST-NET with protocol next hop PNH-IPv6.
+
+6.  DUT is configured with an IPv4 GUE tunnel with destination IPv4-DST-GUE
+    without any explicit tunnel Type of Service (ToS) or Time to Live (TTL)
+    values.
+
+7.  DUT is configured with the following static routes:
+
+    -   To PNH-IPv4, next hop is the statically configured IPv4 GUE tunnel.
+    -   To PNH-IPv6, next hop is the statically configured IPv4 GUE tunnel.
+
+### PF-1.3.1: IPv4 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+ATE action:
+
+*   Generate 12000000 **IPv4 packets** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET ensuring that there are at least 65000 different 5-tuple flows.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of all packets to *10*.
+
+Verify:
+
+*   Policy forwarding rule `matched-packets` counters equals the number of
+    packets generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets:
+    *   Egressing DUT:Port2, DUT:Port3, DUT:Port4 and DUT:Port5.
+    *   Received on ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5.
+*   All packets received on ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5 are
+    GUE encapsulated.
+*   ECMP hashing works over the two LAG interfaces with a tolerance of 6%.
+*   LAG hashing works over the two Singleton ports within LAG1 and LAG2 with a
+    tolerance of 6%.
```

Comment:
How is the 6% arrived at here as a reasonable threshold?

Reply:
This is based on prior testings. However, if there's a different threshold we need to use, please let me know and I'll update accordingly. Thanks.